script/Arcueid/Day 7/07_00_ARC07_3_1_3.txt

```diff
@@ -632,6 +641,7 @@ That said, it's not as though each and every one of them them has it out for hum
 [sha:a032975dc854c46f8c1c111510704f358ff3ddfc]{
 -- Page 616, Offset 18126.
 -- 「『二十七祖』というのは在り方を示す呼称だから変わらないけど、実際にはえーと……残り二十一祖かな？
+// Group PR Mark : Sentence isn't really correct, needing a recheck on TL
```

Comment:
IIRC something about how there's always 27 because that's how many principles there are
> The phrase 'The Twenty-Seven Ancestors' is more a reference to the way things %{i}ought%{/i} to be than the way things %{i}are%{/i}, in reality there are only, um...

Reply:
The problem on this line was the idea that it was a moniker thats just an artifact of a dying society. Instead, it is decoupled from the number of Ancestors and instead refers to the number of Principles.
"在り方を示す呼称だから変わらない"

I think your line works